codes/classical/q-ary_digits/gabidulin.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: gabidulin
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Gabidulin or rank code'
+introduced: '\cite{manual_citation_minilatex:{E. M. Gabidulin, "Theory of Codes with Maximum Rank Distance", Problemy Peredachi Informacii, Volume 21, Issue 1, \emph{3–16} (1985)}}'
+
+description: 'Error-correcting codes over rank metric.'
```

Comment:
Start with "Also called Gabidulin or matrix rank-metric code." Let's unpack the next sentence, in particular "over rank metric". What is a rank metric and how does it feature in the code construction? Is there a relation between its parameters and d? Check Def. 11.1.3 in Concise encyclopedia of coding theory. A full-blown technical definition is not necessary, but instead something that gets the spirit across.

Reply:
Updated